components/Filter/Index.js

```diff
@@ -0,0 +1,145 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { theme } from "@/styles";
+
+export default function Filter({ onSubmit, showFilterWindow }) {
+  const [author, setAuthor] = useState("");
+  const [category, setCategory] = useState("");
+
+  const handleAuthorChange = (event) => {
+    const value = event.target.value;
+    setAuthor(value);
+  };
+
+  const handleCategoryChange = (event) => {
+    const value = event.target.value;
+    setCategory(value);
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    onSubmit({ author, category });
+  };
+
+  return (
+    <StyledFilterForm show={showFilterWindow} onSubmit={handleSubmit}>
+      <StyledFilterBox>
+        <h2>Filter</h2>
+        <StyledCloseButton onClick={() => (window.location.href = "/")}>
+          x
+        </StyledCloseButton>
+        <StyledUl>
+          <StyledFilterLabel>Author: </StyledFilterLabel>
+          <StyledFilterInput
```

Comment:
you should connect your labels and inputs using `htmlFor` in the label and an `id` in the input